lamarr/roster.json

```diff
@@ -102,6 +102,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "watch?v=z0hCl9MUL0Y"
```

Comment:
Can you please update from `"url": "watch?v=z0hCl9MUL0Y" to`
`"url": "z0hCl9MUL0Y"`